modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/add_site_navigation_menu_item.jsp

```diff
@@ -93,7 +93,7 @@ renderResponse.setTitle(LanguageUtil.format(request, "add-x", siteNavigationMenu
 					return;
 				}
 
-				fetch(
+				Liferay.Util.fetch(
```

Comment:
`Liferay.Util.fetch` adds `credentials: 'include'`, so we can remove it from usages.